projects/igniteui-angular/src/lib/grids/grid/grid.component.ts

```diff
@@ -542,103 +523,6 @@ export class IgxGridComponent extends IgxGridBaseDirective implements GridType,
         this._focusIn.emit();
     }
 
-    /**
-     * Sets a list of key-value pairs [row ID, expansion state].
-     * ```typescript
-     * const states = new Map<any, boolean>();
-     * states.set(1, true);
-     * this.grid.expansionStates = states;
-     * ```
-     *
-     * Two-way data binding.
-     * ```html
-     * <igx-grid #grid [data]="data" [(expansionStates)]="model.expansionStates">
-     * <ng-template igxGridDetail let-dataItem>
-     * <div *ngIf="dataItem.Category">
-     *  <header>{{dataItem.Category?.CategoryName}}</header>
-     * <span>{{dataItem.Category?.Description}}</span>
-     * </div>
-     * </ng-template>
-     * </igx-grid>
-     * ```
-	 * @memberof IgxGridComponent
-     */
-    public set expansionStates(value) {
-        this._expansionStates = new Map<any, boolean>(value);
-        this.expansionStatesChange.emit(this._expansionStates);
-        if (this.gridAPI.grid) {
-            this.cdr.detectChanges();
-            this._focusActiveCell();
-        }
-    }
-
-   /**
-     * Expands all master rows.
-     * ```typescript
-     * this.grid.expandAll();
-     * ```
-	 * @memberof IgxGridComponent
-    */
-    public expandAll() {
-        const expandedStates = this.expansionStates;
-        this.data.forEach((rec) => {
-            expandedStates.set(this.primaryKey ? rec[this.primaryKey] : rec, true);
-        });
-        this.expansionStates = expandedStates;
-    }
-
-   /**
-     * Collapses all master rows.
-     * ```typescript
-     * this.grid.collapseAll();
-     * ```
-	 * @memberof IgxGridComponent
-    */
-    public collapseAll() {
-        this.expansionStates = new Map<any, boolean>();
-    }
-
-    /**
-     * Expands the master row by its id. ID is either the primaryKey value or the data record instance.
-     * ```typescript
-     * this.grid.expand(rowID);
-     * ```
-	 * @memberof IgxGridComponent
-     */
-    public expand(rowID: any) {
```

Comment:
Because the base API is expand*Row* and collapse*Row* these need to be migrated.